<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_newDocument">
          <property name="text">
           <string>新增文献</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_editDocument">
          <property name="text">
           <string>修改文献</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_deleteDocument">
          <property name="text">
           <string>删除文献</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineedit_searchText"/>
          </item>
          <item>
           <widget class="QComboBox" name="cbbox_searchTerm"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_newTerm">
            <property name="text">
             <string>新增查询项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clearTerm">
            <property name="text">
             <string>清空查询项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_search">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_document"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
